src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -401,6 +505,57 @@ public Task GetAll()
             }
         }
 
+        [RavenAction("/admin/certificates/export", "GET", AuthorizationStatus.ClusterAdmin)]
+        public Task GetClusterCertificates()
+        {
+            if (IsClusterAdmin() == false)
+                throw new InvalidOperationException("Cannot export certificate collection. This is a 'Cluster Admin' only operation.");
```

Comment:
Why this check, this is already done in the attribute, no?

Reply:
yes